geoportailv3/static/js/mymaps/mymapsdirective.js

```diff
@@ -450,6 +450,10 @@ app.MymapsDirectiveController.prototype.openChooseMapModal = function() {
       } else if (!goog.array.isEmpty(mymaps)) {
         this.choosing = true;
         this.maps = mymaps;
+      } else if (goog.array.isEmpty(mymaps)) {
```

Comment:
Is there an other case you are thinking of?
If not, I would use `else` with no condition.